pkg/cel/ext/kuadrant.go

```diff
@@ -110,3 +111,36 @@ func (l kuadrantLib) ProgramOptions() []cel.ProgramOption {
 func (kuadrantLib) LibraryName() string {
 	return "kuadrant.cel.ext.kuadrant"
 }
+
+func getRegistryWithTypes() *types.Registry {
+	registry, _ := types.NewRegistry(
+		// common.proto
+		&v0.Metadata{},
```

Comment:
Should these all be `v1` _of Gateway API types_?

Reply:
Can you elaborate, please? 
Those meant to be the results of `.proto` message definitions (the ones that implement the `proto.Message` interface) as the registry is used to populate types for our CEL env. 
Also, the `cel.SomethingBinding()` should be enforcing the `ref.Val` type and we use this registry to convert into that :thinking:

Reply:
I'm saying, should we move these from:

```golang
	v0 "github.com/kuadrant/kuadrant-operator/pkg/extension/grpc/v0"
```

to `v0` (both the package _and_ its location) to whatever version this represent? I guess we can't use the dot (`.`), so something like if this is version 1.2 of the Gateway API, to `v1_2` or something?

Reply:
moved to v1 all the new things. I also copied what we had in `v0/kuadrant.proto` to `v1/kuadrant.proto`. Was thinking to leave it, but decided not to: want to keep all imports in the repository (the `google/...` folders). 
Don't know if it matters, but manually copying the `import` folders should allow anyone to regenerate `.pb` files